Gemfile

```diff
@@ -4,13 +4,8 @@ source 'https://rubygems.org'
 
 ruby '2.5.3'
 
-DECIDIM_VERSION = '~> 0.16.0'
+DECIDIM_VERSION = '~> 0.17.0'
```

Comment:
0.17.1 is already out

Reply:
When I try to update to `0.17.1` I get the following:
```console
$ bundle update decidim
Could not find gem 'decidim-dev (= 0.17.1)' in any of the gem sources listed in your Gemfile.
```

Reply:
@isaacmg410 has already notified @oriolgual about this